src/python/WMCore/Storage/DeleteMgr.py

```diff
@@ -130,35 +140,34 @@ def initialiseOverride(self):
         Extract and verify that the Override parameters are all present
 
         """
-        overrideConf = self.overrideConf
-        overrideParams = {
+        self.overrideConf = {
             "command": None,
             "option": None,
             "phedex-node": None,
             "lfn-prefix": None,
         }
 
         try:
-            overrideParams['command'] = overrideConf['command']
-            overrideParams['phedex-node'] = overrideConf['phedex-node']
-            overrideParams['lfn-prefix'] = overrideConf['lfn-prefix']
+            self.overrideConf['command'] = self.overrideParams['command']
```

Comment:
The change of name is not strictly required. It is because these `overrideParams` and `overrideConf` actually confused me too and finally I decided to draw a clear line between them. overrideParams holds any override parameter while overrrideConf is the final, full configuration with some values modified by overrideParams. Ok let me convert them back to what they were.